book/scheduling.md

```diff
@@ -287,20 +281,6 @@ thread to stop at that line of code. When more work comes in to do, such as in
 `schedule_task`, a call to `notify_all` will wake up the thread that called
 `wait`.
 
-It's important to call `wait` at the end of the `run` loop if there is nothing
-left to do. Otherwise that thread will tend to use up a lot of the CPU,
-plus constantly be acquiring and releasing `condition`. This busywork not only
-slows down the computer, but also causes the callbacks from the `Timer` to
-happen at erratic times, because the two threads are competing for the
-lock.[^try-it]
-
-[condition-variable]: https://docs.python.org/3/library/threading.html#threading.Condition
-
-[lock-class]: https://docs.python.org/3/library/threading.html#threading.Lock
-
-[^try-it]: Try removing this code and observe. The timers will become quite
-erratic.
-
```

Comment:
I found this confusing when reading; it makes much more sense _after_ the following code block, so I moved it.